client/madison.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/madison.css" />
+    
+<title>Madison Hudson</title>
+<style>
+    body {
+      background-image: url('/images/madisonbackground.jpg');
+      background-repeat: no-repeat;
+      background-size:cover;
+
+    }
+    </style>
+  </head>
+  <body>
+    <img src="/images/madison.jpg" alt="Photo of Madison" width="500" height="500" class="center">
+
+      <p1 class="paragraph"style="background-color:white;">Hi, I'm Madison! About a year and a half ago I decided I was ready to learn to code after graduating with a business degree and working in customer service. I attended a coding bootcamp in Durham that ended in December 2019. We learned to build full stack applications focusing on Javascript and since then I've been working with C# and .NET as contractor for a maritime company. My hobbies include cooking new recipes, exploring local trails, and watching way too much 90 Day Fiance.<p1>
+
+    
+
```

Comment:
Just format this: remove empty lines, fix paragraph indentation. Also should be just `<p>`, not `<p1>`